src/font/face/coretext.zig

```diff
@@ -343,13 +343,12 @@ pub const Face = struct {
         } = if (!self.isColorGlyph(glyph_index)) .{
             .color = false,
             .depth = 1,
-            .space = try macos.graphics.ColorSpace.createDeviceGray(),
-            .context_opts = @intFromEnum(macos.graphics.BitmapInfo.alpha_mask) &
-                @intFromEnum(macos.graphics.ImageAlphaInfo.only),
+            .space = try macos.graphics.ColorSpace.createNamed(.linearGray),
+            .context_opts = @intFromEnum(macos.graphics.ImageAlphaInfo.only),
         } else .{
             .color = true,
             .depth = 4,
-            .space = try macos.graphics.ColorSpace.createDeviceRGB(),
+            .space = try macos.graphics.ColorSpace.createNamed(.displayP3),
```

Comment:
@qwerasd205 *suggestion*: Should we make this `...ColorSpace.createFromName(.displayP3)`?

Reply:
Why? I could understand changing it to `createWithName` if we wanted to match the name of the CoreGraphics function, but this really feels like splitting hairs either way. I'm not devoted to the name, but if I changed it I'd be more likely to change it to `createWithName` than `createFromName` unless there's particularly good reasoning...

Reply:
@qwerasd205 The way I see it, we are creating a `ColorSpace` from a `ColorSpace.Name` called `.displayP3`.

From that perspective, it would logically flow into `ColorSpace.createFromName(.displayP3)`.

I do think `createWithName` is also a great option.

Reply:
Technically our `macos` pkg is a thin layer over Carbon/AppKit APIs and we just mirror their names, so _technically_ the correct name here is `createWithName` but I'm not going to lose sleep over it and don't really care.

Reply:
@mitchellh @qwerasd205 Im also happy if we leave it the way it is. 

Resolving comment.